ls1intum/Artemis
# Bugfix: Translation for System Notifications

### Checklist

### Motivation and Context
Some translations where missing when working with System Notifications.

### Description
- Properly format translation keys for System Notifications
- Undo Hotfix from 99cb5df5

### Steps for Testing
1. Navigate to System Notifications
3. Create/Edit/Delete Notifications and see translated Alerts
